tasks/main.yml

```diff
@@ -1,27 +1,29 @@
 ---
-- name: Enable backports repository
-  apt_repository:
-    repo: deb http://ftp.debian.org/debian jessie-backports main
-    state: present
+  - apt: update_cache=yes cache_valid_time=3600
+    become: yes
```

Comment:
Same question regarding `become:` as above. Also, why do we update the APT cache both in `main.yml` and in `debian-stretch.yml` / `debian-jessie.yml`? Can we remove one of these tasks?